vc_zoom/indico_vc_zoom/plugin.py

```diff
@@ -160,6 +160,7 @@ def init(self):
         self.inject_bundle('main.js', WPSimpleEventDisplay)
         self.inject_bundle('main.js', WPVCEventPage)
         self.inject_bundle('main.js', WPVCManageEvent)
+        self.inject_bundle('main.js', WPConferenceDisplayBase)
```

Comment:
Do you remember if there was a reason to use the base class instead of `WPConferenceDisplay`? By default bundles are also injected in subclasses, which results in the same bundle being included twice on the actual conference videoconference page.

I think regardless of that we should fix that in the core, ie. deduplicate injected bundles or even do it like we do for core bundles in `_resolve_bundles`  (throwing an exception while in debug mode)

Reply:
I've changed it to `WPConferenceDisplay` and I think there is no problem. I'll open another PR.